apidoc/Annotation.yml

```diff
@@ -76,6 +76,131 @@ properties:
     type: Titanium.UI.View
     since: "3.1.0"
 
+  - name: showMarker
+    summary: |
+        Boolean to show an annotation view that displays a balloon-shaped 
+        marker at the designated location.
+    description: |
+        Must be set during creation. This is ignored if the customView property is set.
+    type: Boolean
+    default: false
+    platforms: [iphone, ipad]
+    osver: {ios: {min: "11.0"} }
```

Comment:
Missing `since: "6.3.0"`. Please also validate and add to other new API's here if missing.

Reply:
Done

Reply:
The property is still there, it should ideally move away completely as suggested in the last review. What do you think?

Reply:
If we make our marker api as you mentioned -
Map.createAnnotation({
latitude: xxxx,
longitude: xxxx,
marker: {
glyphTextColor: 'red',
...
}
});
If developer want to set the glyphTextColor (e.g) at run , it will not be convenient. It is not necessary to set all marker properties at creation only. 
And there are certain conditions, only after that refreshing of annotations happen e.g (delegate!=nil && [delegate viewAttached]). So I think current implementation is fine. What you think ?